addons/website_sale/tests/test_website_sale_pricelist.py

```diff
@@ -303,7 +303,7 @@ class TestWebsitePriceListAvailableGeoIP(TestWebsitePriceListAvailable):
     def setUp(self):
         super(TestWebsitePriceListAvailableGeoIP, self).setUp()
         # clean `property_product_pricelist` for partner for this test (clean setup)
-        self.env['ir.property'].search([('res_id', '=', 'res.partner,%s' % self.env.user.partner_id.id)]).unlink()
+        self.env['ir.property'].sudo().search([('res_id', '=', 'res.partner,%s' % self.env.user.partner_id.id)]).unlink()
```

Comment:
Isn't this `env` with the `SUPERUSER_ID`?  If so, the `sudo()` just looks like noise.